layers/meta-resin-raspberrypi/recipes-connectivity/bluez5/bluez5_%.bbappend

```diff
@@ -7,3 +7,40 @@ pkg_postinst_${PN}_append_raspberrypi3 () {
 	fi
 	systemctl $OPTS mask brcm43438.service
 }
+
+# apply the same patches we apply for the rpi3. this makes bluetooth on
+# rpi0-wireless actually work.
+# The following was copied from meta-raspberrypi/ - keep it in sync.
+pkg_postinst_${PN}_append_raspberrypi () {
+	if [ -n "$D" ]; then
+		OPTS="--root=$D"
+	fi
+	systemctl $OPTS mask brcm43438.service
+}
+
+FILESEXTRAPATHS_prepend := "meta-raspberrypi/recipes-connectivity/bluez5/${PN}:"
```

Comment:
The default PATH should include this already from the bbappend in the BSP layer. Any reason to have it in?